doc/changelog.md

```diff
@@ -255,18 +396,29 @@
   shot in a shots-based execution mode.
   [(#731)](https://github.com/PennyLaneAI/catalyst/pull/731)
 
+* The Catalyst frontend now supports Python logging through PennyLane's `qml.logging` module.
+  [(#660)](https://github.com/PennyLaneAI/catalyst/pull/660)
```

Comment:
Is this feature documented somewhere that we can link to?

Reply:
I couldn't find anything 🤔 @mlxd?

Reply:
Looks like we do: https://docs.pennylane.ai/en/stable/introduction/logging.html

Reply:
cbacdec3a0dfe23bd18251c66625f68c8184d063